docs/introduction-to-smart-contracts.rst

```diff
@@ -181,21 +181,19 @@ arguments ``from``, ``to`` and ``amount``, which makes it possible to track
 transactions.
 
 To listen for this event, you could use the following
-JavaScript code, which uses `web3.js <https://github.com/web3/web3.js/>`_ to create the ``Coin`` contract object,
+JavaScript code, which uses `ethers.js <https://github.com/ethers-io/ethers.js>`_ to create the ``coin`` contract object,
```

Comment:
```suggestion
JavaScript code, which uses `ethers.js <https://github.com/ethers-io/ethers.js>`_ to create the ``Coin`` contract object,
```
Here `Coin` refers to the contract name, not the variable name in the example.

Reply:
> Here `Coin` refers to the contract name, not the variable name in the example.

Got it, thanks for clarifying!